### PR TITLE
changed caption

### PR DESCRIPTION
**Pull request by BKPuma**

### CODE REVIEW

**Review by BSRV1987 — COMMENTED** [review]
All Good

**Review by CarolineChiari — APPROVED** [review]
This is awesome! Great job! very inspiring!

File: _slides/06-BKPuma.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Bastian
```
Comment (dschingisab) [inline _slides/06-BKPuma.md:6]: not good...but what can we do

**Review by dschingisab — APPROVED** [review]
zufriedenstellend

**Pull request merged**